src/app/shared/store/user/auth.effects.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { tap, switchMap, map } from 'rxjs/operators';
+
+import { AuthCustomersService } from '../../auth-customers.service';
+import { AuthActionTypes, LogIn, LogInSuccess } from './auth.actions';
+
+
+@Injectable()
+export class AuthEffects {
+
+  constructor(
+    private actions: Actions,
+    private authService: AuthCustomersService,
+    private router: Router,
+  ) {}
+
+  @Effect()
+  LogIn: Observable<any> = this.actions
+    .pipe(
+      ofType(AuthActionTypes.LOGIN),
+      map((action: LogIn) => action.payload),
+      switchMap(payload => {
+        console.log('Payload ',payload)
+        return this.authService.login(payload)
+          .pipe(
+            map((user) => {
+              return new LogInSuccess(user);
+            })
```

Comment:
Better to return result of `this.authService.login(payload)` from this `switchMap` and then add one more `map` (for returning `new LogInSuccess(user)`) and `catchError` operator if needed